authui-container/README.md

```diff
@@ -0,0 +1,642 @@
+  the **LOGS** tab.
+- While customizing the UI configuration. The modified configuration can be
+  added as an environment variable `UI_CONFIG`. Use the `/admin` panel to
+  modify the configuration. This provides some validation before allowing
+  admins the ability to copy the configuration.
+- To disable the admin panel after customization. To avoid exposing the
+  `/admin` panel or other administrative APIs, a revision can be deployed
+  with the environment variable `ALLOW_ADMIN` set to `false` or `0`.
+
+## Advanced
+
+### Customizing Auth UI Domain
+
+The Cloud Run provisioned URL for the Auth UI (eg.
+`servicename-6tihbjyvla-uc.a.run.app`) can be customized (`auth.example.com`)
+by following these instructions:
+
+- Follow the
+  [instructions](https://cloud.google.com/run/docs/mapping-custom-domains) to
+  map the custom domain (eg. `auth.example.com`).
+- Update the authentication UI URL in the IAP panel for the resource that was
+  using it (by replacing the existing domain with the custom domain) and save
+  it.
+- Whitelist the new domain with Identity Platform in the settings &gt; security
+  section and save it.
+
+### Using the same hosted Auth UI for multiple IAP resources
+
+The same Cloud Run service deployed for one IAP resource can be shared among
+multiple IAP resources in the same project even if they have different Identity
+Platform configurations. This works as follows:
+
+- Enable IAP with one resource and deploy the authentication UI with Cloud Run
+  as documented above.
+- Enable IAP with another resource. In the authentication URL input, the same
+  previous authentication URL (provisioned by Cloud Run) should be entered.
+- Since the second resource may have a different Identity Platform configuration
+  (different associated tenants / enabled IdPs), a new revision of the Cloud Run
+  service will need to be deployed. Redeploying a revision of the instance
+  should automatically pick up the additional GCIP settings. Since the default
+  configuration is lazy loaded on first authentication UI web server access,
+  this step may be not required if the first visit occurs after the second
+  resource is configured.
+
+## Contributions
+
+The following instructions illustrate how to make contributions to the
+hosted authentication UI used by IAP and how to generate and deploy your own
+custom container image build.
+
+### Installation
+
+To set up a development environment to build the sample from source, you must
+have the following installed:
+- Node.js (>= 10.0.0)
+- npm (should be included with Node.js)
+- [Google Cloud SDK](https://cloud.google.com/sdk/) which includes the gcloud
+  command line tool.
+
+Download the authentication UI container source code:
+
+```bash
+git clone https://github.com/GoogleCloudPlatform/iap-gcip-web-toolkit.git
+```
+
+Install the authentication UI container app:
+
+```bash
+cd iap-gcip-web-toolkit/sample/authui-container
```

Comment:
should this be "cd iap-gcip-web-toolkit/authui-container"

Reply:
Great catch. I fixed it.